src/pages/SignIn.js

```diff
@@ -1,15 +1,21 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React, {useContext} from 'react';
+import {Link} from 'react-router-dom';
+import {AuthContext} from "../context/AuthContext";
 
 function SignIn() {
+    const {inlogFunction} = useContext(AuthContext)
+
```

Comment:
In de uitwerkingen staat dat je bij de form de preventDefault er nog bij kan zetten. Dit is de functie die je hier dan declareert:
  function handleSubmit(e) {
    e.preventDefault();
    login();
  }